docs/plugins/repository-s3.asciidoc

```diff
@@ -215,8 +215,14 @@ include a port. For example, the endpoint may be `172.17.0.2` or
 https://minio.io[Minio] is an example of a storage system that provides an
 S3-compatible API. The `repository-s3` plugin allows {es} to work with
 Minio-backed repositories as well as repositories stored on AWS S3. Other
-S3-compatible storage systems may also work with {es}, but these are not tested
-or supported.
+S3-compatible storage systems may also work with {es}, but these are not
+covered by the {es} test suite.
+
+Note that some storage systems claim to be S3-compatible without correctly
+supporting the full S3 API. The `repository-s3` plugin requires full
+compatibility with S3. If you wish to use another storage system with the
```

Comment:
```suggestion
compatibility with S3. In particular it must support the same set of API
endpoints, return the same errors in case of failures, and offer a consistency
model no weaker than S3's when accessed concurrently by multiple nodes. If you
wish to use another storage system with the
```